src/Microsoft.DocAsCode.Common/Loggers/LoggerPhaseScope.cs

```diff
@@ -26,14 +21,13 @@ public LoggerPhaseScope(string phaseName, bool enablePerformanceScope)
             _originPhaseName = GetPhaseName();
             phaseName = _originPhaseName == null ? phaseName : _originPhaseName + "." + phaseName;
             SetPhaseName(phaseName);
-            if (enablePerformanceScope)
-            {
-                _performanceScope = new PerformanceScope("Scope:" + phaseName, LogLevel.Diagnostic);
-            }
         }
 
+        public LoggerPhaseScope(string phaseName, bool enablePerformanceScope)
+            : this(phaseName, LogLevel.Diagnostic) { }
```

Comment:
Obsolete first, then remove it.

---
In reply to: [95927157](https://github.com/dotnet/docfx/pull/1168#discussion_r95927157) [](ancestors = 95927157)